README.md

```diff
@@ -42,12 +42,15 @@ jobs:
           GIST_ID: 123abc
 ```
 
-2. 💰 Profit
+2. Create a [secret](https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets) 🔑 by going to **GitHub repo > Settings > Secrets > New secret** with the following:
```

Comment:
Tiny tweak:

```suggestion
2. [Create a secret](https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets) 🔑 by going to **GitHub repo > Settings > Secrets > New secret** with the following:
```

Reply:
This tiny tweak makes it look so much better.